### PR TITLE
https://issues.redhat.com/browse/ACM-6223 Update create_infra_env.adoc

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-6223

If this is the correct link to use, I need to update the link in 2.6

### CODE REVIEW

**Comment by dockerymick** [comment]
I have made the similar updates that were made in the [PR for 2.6](https://github.com/stolostron/rhacm-docs/pull/5261/files). However, I would like to receive confirmation that I am pointing to the correct OCP doc.

File: clusters/cluster_lifecycle/create_infra_env.adoc [hunk 1]
```diff
@@ -10,9 +10,10 @@ You can use the console to create an infrastructure environment to manage your h
...
-- Late binding: Enable the host to be booted by an infrastructure administrator. The creator of a cluster can bind a cluster to that host at a later time. The cluster creator does not have to have administrator privileges to the infrastructure when using late binding.
+//I received a 404 with this link. Checked the 4.12 docs
+//Seems like this is the correct link: https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html-single/scalability_and_performance/index#installing-disconnected-rhacm_ztp-preparing-the-hub-cluster[Installing GitOps ZTP in a disconnected environment 
+- Zero-touch provisioning of clusters: Deploy clusters using a script. See https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html-single/scalability_and_performance/index#installing-disconnected-rhacm_ztp-preparing-the-hub-cluster[Installing GitOps ZTP in a disconnected environment] in the {ocp} documentation for more information.
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/create_infra_env.adoc]:
```suggestion
- Zero-touch provisioning of clusters: Deploy clusters using a script. See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html-single/scalability_and_performance/index#installing-disconnected-rhacm_ztp-preparing-the-hub-cluster[Installing GitOps ZTP in a disconnected environment] in the {ocp} documentation for more information.
```

**Review by oafischer — APPROVED** [review]
Link works for me as well, thanks for fixing!

This conversation might also be of interest to you: https://redhat-internal.slack.com/archives/GVDCLR7A5/p1686082066052749

**Comment by dockerymick** [comment]
@oafischer based on the thread that you linked, are you suggesting that I change my reference link in this PR or update the thread and let them know which link i used?

**Comment by oafischer** [comment]
I'm not suggesting any changes to the PR, I just linked that conversation because it provides background on why the link to that specific section in the OCP docs changed. It's also confirmation from Laura on where to find the general ZTP docs in the OCP docs at the moment, so indirect link confirmation.

**Comment by dockerymick** [comment]
@oafischer word! Thanks for confirming and thanks for the background info ! Merging this PR

**Pull request merged**